packages/connect-fastify/src/fastify-connect-plugin.ts

```diff
@@ -60,30 +60,21 @@ export function fastifyConnectPlugin(
     done();
     return;
   }
+  if (opts.acceptCompression === undefined) {
+    opts.acceptCompression = [compressionGzip, compressionBrotli];
+  }
```

Comment:
Defaults for acceptCompression moved a couple of lines up. Otherwise, createConnectRouter overrides with an empty array as a default, and these defaults never apply.